packages/replay/src/types.ts

```diff
@@ -48,9 +48,11 @@ export interface WorkerResponse {
   id: number;
   method: string;
   success: boolean;
-  response: ReplayRecordingData;
+  response: unknown;
```

Comment:
Why this change?

Reply:
It doesn't always return `ReplayRecordingData` (e.g. in `addEvent`)